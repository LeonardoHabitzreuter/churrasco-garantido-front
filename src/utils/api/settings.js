export const BASE_API_URL = process.env.apiHost || 'http://localhost:3001'
