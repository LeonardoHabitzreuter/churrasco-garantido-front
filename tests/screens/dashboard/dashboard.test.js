import React from 'react'
import { shallow } from 'enzyme'
import Dashboard from '../../../src/screens/dashboard'
import api from '../../../src/utils/api'
api.getUser = jest.fn()

describe('Dashboard component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<Dashboard />)
    expect(wrapper).toMatchSnapshot()
  })
})
