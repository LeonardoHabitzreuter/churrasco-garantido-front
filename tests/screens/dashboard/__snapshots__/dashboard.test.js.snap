// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component should renders properly 1`] = `
<div
  className="col-sm-8"
>
  <PageHeader
    bsClass="page-header"
  >
    Minhas empresas
  </PageHeader>
  <Loader
    loading={true}
    size="large"
  />
  <Alert
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style=""
  />
  <Table
    columns={
      Object {
        "cnpj": "CNPJ",
        "name": "Nome fantasia",
        "ordersQuantity": "QTD pedidos",
      }
    }
    lines={Array []}
  />
</div>
`;
